package.json

```
@@ -11,11 +11,13 @@
     "new-version": "lerna version --conventional-commits --create-release github --yes -m \"chore(release): %v [ci skip]\"",
     "publish": "lerna publish from-package --yes",
     "lerna": "lerna",
+    "dd-approve": "node dependabot-approve.js",
```

```suggestion
    "dependabot-approve": "node dependabot-approve.js",
```

`dd` means nothing to me, so let's be clear what it's doing